agent/app/agent.go

```diff
@@ -134,6 +135,7 @@ type ecsAgent struct {
 	stateManagerFactory         factory.StateManager
 	saveableOptionFactory       factory.SaveableOption
 	pauseLoader                 pause.Loader
+	appNetLoader                serviceconnect.Loader
```

Comment:
nit: appNetLoader should be called serviceConnectLoader to match package naming

Reply:
or maybe `serviceconnectLoader` to be consistent with serviceconnectManager and serviceconnectRelay below